docs/index.md

```diff
@@ -1,19 +1,20 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: BobTheBodyBuilder
 ---
 
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
-[![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
+[![CI Status](https://github.com/AY2122S1-CS2103T-W16-2/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S1-CS2103T-W16-2/tp/actions)
+[![codecov](https://codecov.io/gh/AY2122S1-CS2103T-W16-2/tp/branch/master/graph/badge.svg?token=PQVA8CSXQ8)](https://codecov.io/gh/AY2122S1-CS2103T-W16-2/tp)
 
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**BobTheBodyBuilder(BBTB) is a desktop application for gym managers to manage their clients and gym bookings.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
```

Comment:
Can we change this to align it with https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/17/files?

Reply:
Yup. Will make the change